docker-compose.yml

```diff
@@ -31,10 +31,12 @@ services:
     ports:
     - '3309:3306'
     volumes:
-    - /var/lib/postgresql/data
+    - db:/var/lib/postgresql/data
```

Comment:
Shouldn't dbtest have its own volume ? It seems one will crush the other.

Reply:
I was testing something, rolled back that